### PR TITLE
DDF-2131: Removed history metacards from ingest results

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do? 
Removes History metacard from the result list on an ingest
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@djblue 
@andrewkfiedler 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire
@pklinef

### CODE REVIEW

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java [hunk 1]
```diff
@@ -51,7 +58,28 @@ public HistorianPlugin(CatalogFramework catalogFramework) {
...
+        if (historyMetacards.isEmpty()) {
+            return input;
+        }
+
+        List<Metacard> resultMetacards = new ArrayList<>(input.getCreatedMetacards());
+        resultMetacards.removeAll(historyMetacards);
+
+        Map<String, Serializable> properties =
+                input.getProperties() != null ? input.getProperties() : new HashMap<>();
+        properties.put(HISTORY_METACARDS, new ArrayList<Serializable>(historyMetacards));
```
Comment (pklinef) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: It would be nice to include the history metacards in update and delete responses too.

Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: That we can do. I'll update tomorrow morning.

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java [hunk 2]
```diff
@@ -43,6 +48,8 @@
 public class HistorianPlugin implements PostIngestPlugin, PreIngestPlugin {
     private static final Logger LOGGER = LoggerFactory.getLogger(HistorianPlugin.class);
 
+    private static final String HISTORY_METACARDS = "history-metacards";
```
Comment (kcwire) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: should this be public so we can reference it in other places?

Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: We could.. but this is a bundle. Should I move it to the HistoryMetacardImpl instead maybe?

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java (hunk 1 above)
Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: @pklinef Updated update and delete to include the history metacards in properties as well, but will put up as a separate PR along with content change.
